### DISCUSSION TITLE
[Solved] how to add parent tag for child tag?.

### DISCUSSION THREAD

**Discussion by dir1337** [kylechui/nvim-surround#267] [General]
**Situation:**
if I have the following code:

> 
```html
 <h1>HTML Table</h1>
    <table>
      <th>col 1</th> 
     |<th>col 2</th>
    </table>
```
My cursor is at <th> tag (as shown) and I want to insert both of the <th> (child) tags inside the <tr> (parent) tag.
**Question**
Any tips for this?

**Comment by kylechui** [comment]
I'm not quite sure what you're trying to do here, could you provide "before" and "after" sample buffers to help me understand what the diff is that you want?

**Reply by dir1337** [reply]
**Before:** 
```html
 <h1>HTML Table</h1>
    <table>
      <th>col 1</th> 
      <th>col 2</th>
    </table>
```
**After:**
```html
 <h1>HTML Table</h1>
    <table>
       <tr>
         <th>col 1</th> 
         <th>col 2</th>
       </tr>
    </table>
```

**Notice:**  
The \<tr> tag added as parent of \<th> tags.

**Question**
Can it be possible to add a parent tag for the "multi-lines" children tags?

**Reply by kylechui** [reply]
If your cursor is on the `table` tag, you could do `ysitt` for "you surround in tag, with a tag".

**Reply by dir1337** [reply]
Yes it work's. 
Advice: A detailed document for this package will be really good for the users.
Thank you.

**Reply by kylechui** [reply]
Have you looked at `:h nvim-surround` and `:h text-objects`?

**Reply by dir1337** [reply]
I have now.
Thank you.
